src/vivarium_csu_alzheimers/components/population.py

```diff
@@ -0,0 +1,157 @@
+import numpy as np
+import pandas as pd
+from vivarium import Component
+from vivarium.framework.engine import Builder
+from vivarium.framework.event import Event
+from vivarium.framework.population import SimulantData
+from vivarium_public_health import utilities
+from vivarium_public_health.population import ScaledPopulation
+
+from vivarium_csu_alzheimers.constants import data_keys
+from vivarium_csu_alzheimers.constants.metadata import ARTIFACT_INDEX_COLUMNS
+
+
+class AlzheimersPopulation(ScaledPopulation):
+    def setup(self, builder: Builder) -> None:
+        super().setup(builder)
+        self.key_columns = builder.configuration.randomness.key_columns
+
+    def on_initialize_simulants(self, pop_data: SimulantData) -> None:
+        if pop_data.user_data.get("sim_state") != "time_step":
+            super().on_initialize_simulants(pop_data)
+            return
+
+        # get pop_data.user_data["demographic_counts"] if it exists
+        if "demographic_counts" in pop_data.user_data:
+            # demographic_counts will be a pd.Series
+            demographic_counts = pop_data.user_data["demographic_counts"]
```

Comment:
I think this a better way to communicate that this will be a Series. We're not running mypy, so this won't break anything.
```suggestion
            demographic_counts: pd.Series = pop_data.user_data["demographic_counts"]
```